### PR TITLE
AutoYaST: describe the <aliases> section

### PR DESCRIPTION
**Pull request by imobachgs**
### PR creator: Description

The `<aliases>` section is missing from the AutoYaST documentation. This PR adds the description and a simple example.

An issue is still pending. As you can see, the elements in that section are case-sensitive. We have improved the implementation and starting with `yast2-network` 4.2.104 (SP2), and 4.3.75 (SP3), and 4.4.25 (SP4) that's not the case anymore. So I am unsure about how to express this information. Maybe the user does not need to know so many details and we could simply remove the notice from the SP4 documentation (it will not be case sensitive from the beginning).

### PR creator: Are there any relevant issues/feature requests?

* [bsc#1190739](https://bugzilla.suse.com/show_bug.cgi?id=1190739)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/ay_networking.xml [hunk 1]
```diff
@@ -755,6 +765,98 @@
    </example>
   </sect2>
 
+   <sect2 xml:id="CreateProfile-Network-Aliases">
+     <title>Assigning Multiple IP Addresses</title>
+
+     <para>
+      &ay; allows assigning multiple IP addresses to the same interface. They are specified using
```
Comment (dmpop) [inline xml/ay_networking.xml:772]:
```suggestion
      &ay; makes it possible to assign multiple IP addresses to the same interface. They are specified using
```

File: xml/ay_networking.xml [hunk 2]
```diff
@@ -755,6 +765,98 @@
...
+      an <literal>aliases</literal> element that contains an <literal>aliasX</literal> entry for
+      each address.
+     </para>
+
+     <para>
+      Each of those entries support the following elements:
```
Comment (dmpop) [inline xml/ay_networking.xml:778]:
```suggestion
      Each entry supports the following elements:
```

File: xml/ay_networking.xml [hunk 3]
```diff
@@ -755,6 +765,98 @@
...
+         Network prefix, for example: <literal>24</literal>
+        </para>
+       </listitem>
+      </varlistentry>
+
+      <varlistentry>
+       <term>NETMASK</term>
+       <listitem>
+        <para>
+         Address' netmask.
```
Comment (dmpop) [inline xml/ay_networking.xml:804]:
```suggestion
         Netmask of the address.
```

File: xml/ay_networking.xml [hunk 4]
```diff
@@ -755,6 +765,98 @@
...
+         <literal>IPADDR</literal> element.
+        </para>
+       </listitem>
+      </varlistentry>
+
+      <varlistentry>
+       <term>LABEL</term>
+       <listitem>
+        <para>
+         Address' label.
```
Comment (dmpop) [inline xml/ay_networking.xml:817]:
```suggestion
         Label of the address.
```

File: xml/ay_networking.xml [hunk 5]
```diff
@@ -755,6 +765,98 @@
...
+       </listitem>
+      </varlistentry>
+     </variablelist>
+
+     <note>
+      <title>Case Sensitive Elements</title>
+      <para>
+       Please, beware that for historical reasons, within the <literal>aliases</literal> section the
+       <literal>IPADDR</literal>, <literal>PREFIXLEN</literal>, <literal>LABEL</literal> and
+       <literal>NETMASK</literal> elements are case-sensitive.
```
Comment (dmpop) [inline xml/ay_networking.xml:828]:
```suggestion
       Keep in mind that for historical reasons, the <literal>IPADDR</literal>, <literal>PREFIXLEN</literal>, <literal>LABEL</literal> and <literal>NETMASK</literal> elements within the <literal>aliases</literal> section are case-sensitive.
```

**Review by dmpop — COMMENTED** [review]
A few suggestions.

**Comment by imobachgs** [comment]
Thanks for the review and the suggestions :-)

**Comment by cjschroder** [comment]
very good! thanks @imobachgs and @dmpop. I will merge and backport.

**Pull request merged**